lib/node_modules/@stdlib/assert/is-cube-number/benchmark/benchmark.js

```diff
@@ -31,56 +31,20 @@ var isCubeNumber = require( './../lib' );
 
 // MAIN //
 
-bench( pkg+'::primitives', function benchmark( b ) {
+bench( pkg+'::primitives,true', function benchmark( b ) {
 	var values;
 	var bool;
 	var i;
 
 	values = [
-		'5',
 		5,
 		4,
 		0.5,
 		1.0,
 		0.0,
 		3.14,
 		-5,
```

Comment:
This all vvalues shall not return true right?

Reply:
Currently, most of the numbers in the list will evaluate as `false`. As such, the list should be updated.